### PR TITLE
Add tests for OBlock

### PR DESCRIPTION
**Pull request by asilenko**
Add test for OBlock's:
- number of rows,
- number of cols,
- dots.

### CODE REVIEW

**Comment by AndriiTrofimovD** [comment]
I think there's a typo, you're testing OBlock, but method names are for YBlock:)

**Comment by asilenko** [comment]
> I think there's a typo, you're testing OBlock, but method names are for YBlock:)

Thank you for review.  I corrected methods names.

**Comment by asilenko** [comment]
I'll wait for 0.4 conflict :-)

File: src/test/java/com/epam/prejap/tetris/block/OBlockTest.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import static org.testng.Assert.*;
+
+/**
+ * @author Anna Silenko
+ */
+
```
Comment (TJB4epam) [inline src/test/java/com/epam/prejap/tetris/block/OBlockTest.java:11]: Please don't do that. It's confusing, the Javadoc seems like it's NOT for the class.

**Review by TJB4epam — APPROVED** [review]

**Pull request merged**